scripts/config.pl

```diff
@@ -166,14 +191,22 @@
 my @config_lines = <$config_read>;
 close $config_read;
 
-my ($exclude_re, $no_exclude_re);
+# Add required baremetal symbols to the list that is included.
+if ( $action eq "baremetal" ) {
+    @non_excluded = ( @non_excluded, @non_excluded_baremetal );
+}
+
+my ($exclude_re, $no_exclude_re, $exclude_baremetal_re);
 if ($action eq "realfull") {
     $exclude_re = qr/^$/;
     $no_exclude_re = qr/./;
 } else {
     $exclude_re = join '|', @excluded;
```

Comment:
Preexisting issue: the regexp should be anchored. As the stand, if an option name happens to be a prefix of another one, it'll be included. This isn't a problem right now, it's outside the goal of this PR, and we're in a hurry, so let's fix it later in https://github.com/ARMmbed/mbedtls/pull/1164/files#r158308208